### ISSUE TITLE
Name the handlers after the `displayName` property if it is present

### ISSUE THREAD

**Issue by robvanmieghem** [Jumpscale/go-raml#33]
Currently the following raml snippet
```raml
/users/{username}/info:
    get:
        displayName: GetUserInformation
```
generates the following handler in go:
```go
// It is handler for GET /users/{username}/info
usernameinfoGet(http.ResponseWriter, *http.Request)
```

It would be nicer if the methodname would be the displayname:
```go
// It is handler for GET /users/{username}/info
GetUserInformation(http.ResponseWriter, *http.Request)
```

**Comment by iwanbk** [comment]
It will make the handler name inconsistent.

**Comment by zaibon** [comment]
That's a feature I would also like to have.
As soon as we will have some more complexe endpoint, name of the method will become really unclean. I think it won't help to keep the code simple and clean.

**Comment by iwanbk** [comment]
```
As soon as we will have some more complexe endpoint, name of the method will become really unclean
```
OK, make sense

**Comment by iwanbk** [comment]
implemented in #35
